scripts/render_nlp_tags.py

```diff
@@ -51,7 +51,8 @@ def render_backbone_table(template):
                 f"{preset} | "
                 f"{format_path(metadata)} | "
                 f"{format_param_count(metadata['params'])} | "
-                f"{metadata['description']} \n"
+                f"{metadata['description']} "
+                f"[Model Card]({metadata['model_card']})\n"
```

Comment:
I'm ok with this, but I wonder if there are other ways we could present this information.

Another idea (not sure if it is a good one). We just rename the second column from `Model` -> `Model Card` and put this link there. And make the preset name in the first column a link the model page.

@fchollet any thoughts on this?

Reply:
Sorry for the delay, I just checked with @fchollet on this. Let's actually go with the original approach you had! Model card append to description sounds good to us. Thanks!